content/geoip/docs/databases/anonymous-ip.mdx

```diff
@@ -134,17 +134,30 @@ These are named `GeoIP2-Anonymous-IP-Blocks-IPv4.csv` and
   </tbody>
 </table>
 
-### Example Files
+### Conversion Utility
+
+<ConversionUtility />
+
+## Example Files
+
+We maintain example files in CSV and MMDB format. The files contain dummy data
+rather than real GeoIP data.
+
+### CSV example files
```

Comment:
Is there a specific reason for the different casing for this heading? For example, `Conversion Utility` vs. `CSV example files`

Reply:
i think miguel has a preference for not using title case but there's no good reason not to use it here because it's used across the dev portal. i will change it in my next commit.